### ISSUE TITLE
An issue when roslaunch dronet_launch.launch

### ISSUE THREAD

**Issue by Dong-Wang1** [uzh-rpg/rpg_public_dronet#21]
Hi, 
My name is Dong, nice to write to you. Well, I got a problem with dronet_launch.launch, to be actual, it is a problem with utils.py file.
Here are the outcomes：
File "/home/leo/fubuki_ws/src/dronet/dronet_perception/src/Dronet/utils.py", line 14, in callback_img
    image_type = data.encoding()
 AttributeError: 'cv2.VideoCapture' object has no attribute 'encoding'
[dronet_perception-2] process has died [pid 15319, exit code 1, cmd /home/leo/fubuki_ws/src/dronet/dronet_perception/nodes/dronet_node.py cnn_predictions:=/cnn_out/predictions state_change:=bebop/state_change camera:=bebop/image_raw __name:=dronet_perception __log:=/home/leo/.ros/log/663c4db6-d7c9-11e8-9bed-7c67a289d2da/dronet_perception-2.log].
log file: /home/leo/.ros/log/663c4db6-d7c9-11e8-9bed-7c67a289d2da/dronet_perception-2*.log

Now it the problem, I run this on my laptop, not in the drone, however, if I use cv2.imread() to import one picture and edit some lines like this: 

**imgpath = "/home/leo/photo1/1479425441182877835.jpg"**

bridge = CvBridge()
 
def callback_img(data, target_size, crop_size, rootpath, save_img):  
 '''      
 try:
        image_type = data.encoding()

        img = bridge.imgmsg_to_cv2(data, image_type)
 except CvBridgeError, e:
        print e
 '''   
        **img = cv2.imread(imgpath)**
        img = cv2.resize(img, target_size)
        img = cv2.cvtColor(img, cv2.COLOR_BGR2GRAY)
        img = central_image_crop(img, crop_size[0], crop_size[1])

Then, it works with this picture, I guess this is because of some version issues, but my opencv-python is 3.1.0, the same with your message in readme.md. Could you please tell me what's wrong with this "encoding" in utils.py?
Thank you!

**Comment by antonilo** [comment]
Hi Dong thanks for your message. The open-cv used for drone control is the cv2bridge available for ROS and not python-opencv. As you correctly noticed, this is simpy a version issue. Please install [cvbridge](http://wiki.ros.org/cv_bridge) and see if the problem persists!

**Comment by Dong-Wang1** [comment]
> Hi Dong thanks for your message. The open-cv used for drone control is the cv2bridge available for ROS and not python-opencv. As you correctly noticed, this is simpy a version issue. Please install [cvbridge](http://wiki.ros.org/cv_bridge) and see if the problem persists!

Yes, that is! I did it, I should learn some about cvbridge before I posted this issue, but anyway, thanks!
